docs/versioned_docs/version-zenith/quickstart/183917-daggerverse.mdx

```diff
@@ -0,0 +1,23 @@
+---
+slug: /quickstart/183917/daggerverse
+hide_table_of_contents: true
+title: "Use the Daggerverse"
+---
+import PartialExperimentalDocs from '../partials/_experimental.mdx';
+
+# Quickstart
+
+## Use the Daggerverse
+
+<PartialExperimentalDocs />
+
+As you've seen in previous sections, modules can be consumed from remote sources, or installed and used locally. The [Daggerverse](https://daggerverse.dev) is an online catalog of Dagger modules that enables users to easily discover and use modules created by the Dagger community.
+
+All Dagger CLI commands work as expected with Daggerverse URLs. Here are a few more examples for you to try:
+
+- [Publish a container to the ttlsh registry](https://daggerverse.dev/mod/github.com/shykes/daggerverse/ttlsh): `dagger call -m github.com/shykes/daggerverse/ttlsh publish --ctr alpine:latest`
+- [Play a game of yamlinvaders](https://daggerverse.dev/mod/github.com/sipsma/daggerverse/yamlinvaders): `dagger -m github.com/sipsma/daggerverse/yamlinvaders shell play`
+- [Stitch videos together](https://daggerverse.dev/mod/github.com/marcosnils/daggerverse/videostitch): `wget -q https://download.samplelib.com/mp4/sample-5s.mp4 && wget -q https://download.samplelib.com/mp4/sample-10s.mp4 && dagger download -m github.com/marcosnils/daggerverse/videostitch@main stitch --src-dir .`
```

Comment:
```suggestion
- [Stitch videos together](https://daggerverse.dev/mod/github.com/marcosnils/daggerverse/videostitch): `wget https://github.com/sipsma/dagger/blob/3ea8e33d91bf54717a355bb46378aa84ebf4b52e/core/schema/module.go#L745-L74://download.samplelib.com/mp4/sample-5s.mp4 && wget -q https://download.samplelib.com/mp4/sample-10s.mp4 && dagger -m github.com/marcosnils/daggerverse/videostitch@main call -o output.mp4 stitch --src-dir .`
```

But I also can't confirm that works yet because I get an error:
`process "ffmpeg -i sample-10s.mp4 -c copy intermediate0.ts" did not complete successfully: exit code: 125 `

Not sure, but exit code 125 makes me think it could *maybe* be some sort of platform specific issue (on a linux vm on arm64). cc @marcosnils

Reply:
This works for me (the original version, since I'm using 0.9.7):

vikram@instance-2:~/test$ wget -q https://download.samplelib.com/mp4/sample-5s.mp4 && wget -q https://download.samplelib.com/mp4/sample-10s.mp4 && dagger download -m github.com/marcosnils/daggerverse/videostitch@main stitch --src-dir .
✔ dagger download stitch [10.02s]
┃ Asset exported to "."                                                                                    
• Engine: 1beb6e9ccc53 (version v0.9.7)
⧗ 20.84s ✔ 103 ∅ 3
vikram@instance-2:~/test$ ls
output.mp4    sample-10s.mp4
sample-5s.mp4 
```

Updated to the new chainable command.

Reply:
Okay cool, I think I might have just not setup qemu emulation, which is a "me problem" technically :-)